vendor/engines/sanger_sequencing/spec/services/sanger_sequencing/sample_result_file_saver_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe SangerSequencing::SampleResultFileSaver do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:facility) { FactoryGirl.create(:facility) }
+  let(:product) { FactoryGirl.build(:service, facility: facility).tap { |s| s.save(validate: false) } }
```

Comment:
I'd prefer having more of this setup into factory traits (though really I wish test setup didn't have to be this elaborate).